includes/admin/class-wc-admin-notices.php

```diff
@@ -391,8 +399,8 @@ public static function add_min_version_notice() {
 	 * @return void
 	 */
 	public static function wp_php_min_requirements_notice() {
-		if ( apply_filters( 'woocommerce_hide_php_wp_nag', get_user_meta( get_current_user_id(), 'dismissed_wp_php_min_requirements_notice', true ) ) ) {
-			self::remove_notice( 'wp_php_min_requirements' );
+		if ( apply_filters( 'woocommerce_hide_php_wp_nag', get_user_meta( get_current_user_id(), 'dismissed_' . WC_PHP_MIN_REQUIREMENTS_NOTICE . '_notice', true ) ) ) {
+			self::remove_notice( WC_PHP_MIN_REQUIREMENTS_NOTICE );
```

Comment:
Instead of going through all the effort and having to update this code with each release we introduce why not just md5 the WC_NOTICE_MIN_PHP_VERSION and  WC_NOTICE_MIN_WP_VERSION concatenated? Might have difficulties with the way the missed noticed is stored though, worth looking at?

Reply:
That's a great suggestion. I don't think we even need md5, just concatenation of `WC_NOTICE_MIN_PHP_VERSION` . `WC_NOTICE_MIN_WP_VERSION` should provide uniqueness. Making the change now.